gluten-core/src/main/scala/io/glutenproject/execution/FileSourceScanExecTransformer.scala

```diff
@@ -278,6 +278,7 @@ class FileSourceScanExecTransformer(@transient relation: HadoopFsRelation,
     selected
   }
 
+  override val nodeNamePrefix: String = "NativeFile"
```

Comment:
Better to use "NativeFileScan"? @FelixYBW @zzcclp

Reply:
Just use "NativeFile" and it will work. After some digging, here is the reason:
"NativeFile" is only the prefix part, there is a nodeName in DataSourceScanExec which contains the "Scan " part.
The final display result will be $nodeNamePrefix$nodeName and others, refer to DataSourceScanExec.simpleString

Reply:
Looks reasonable.